### PR TITLE
JA: add js/tutorials/index.md

### PR DESCRIPTION
**Pull request by technohippy**
I translated below into Japanese.
https://github.com/tensorflow/tfjs-website/blob/devsite/docs/tutorials/index.md

### CODE REVIEW

**Comment by technohippy** [comment]
I signed with another email.

File: site/ja/js/tutorials/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+TensorやOptimizerのような低レベルな詳細については考えずに機械学習を始めたい？
+
+TensorFlow.jsを土台として作成されたml5.jsライブラリを利用すると、簡潔でわかりやすいAPIを通じてブラウザ上で機械学習のアルゴリズムとモデルを使用できます。
+
+<a class="button button-white" href="https://ml5js.org">ml5.jsを取得</a>
+
+
+## TensorFlow.jsを準備する
+
+テンソルやレイヤー、オプティマイザ、損失関数などの概念に馴染みがある？（もしくはそれらに詳しくなりたい？） TensorFlow.jsでは JavaScriptでニューラルネットワークプログラミングを行うための柔軟なビルディングブロックが利用できます。
```
Comment (sfujiwara) [inline site/ja/js/tutorials/index.md]: [weak proposal]
I guess `ビルディングブロック ==> パーツ` is more natural for Japanese.
But, it a weak proposal and you can choose one you prefer.

File: site/ja/js/tutorials/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# はじめに
+
+TensorFlow.jsは ブラウザとNode.js内で 機械学習モデルの訓練とデプロイを行うための JavaScriptライブラリです。
```
Comment (sfujiwara) [inline site/ja/js/tutorials/index.md]: It better to insert white space between alphabetical word and Hiragana (or Katakana).
We are currently discussing on translation guideline for Japanese.
How do you think about it?

**Comment by lamberta** [comment]
Thanks for reviewing. For a style guide for the Japanese community translations, feel free to add to the  [ja/README.md](https://github.com/tensorflow/docs/blob/master/site/ja/README.md) file, or discuss on the docs-ja@tensorflow.org list. If you think the translation guide will be longer than should go in the readme, we can add a `ja/translation_guide.md` file if  you think that helps.

Let me know how using RedPen goes. We're not ready to include it into the docs repo, yet, but might be worth showing the other community translation groups if you think it's useful. Thanks

File: site/ja/js/tutorials/index.md (hunk 2 above)
Comment (technohippy) [reply site/ja/js/tutorials/index.md]: The space between an alphabetic word and a Japanese word should be the issue of style. I don't think it's good idea to insert white space between them by hand.

File: site/ja/js/tutorials/index.md (hunk 1 above)
Comment (technohippy) [reply site/ja/js/tutorials/index.md]: Now I used 構成要素 instead of ビルディングブロック. What do you think about it?

**Pull request merged**